image/templates/helm/shared/templates/_storage_classes.tpl

```diff
@@ -0,0 +1,36 @@
+{{/*
+  srox.getStorageClasses
+   */}}
+{{- define "srox.getStorageClasses" -}}
+
+  {{- $ := index . 0 -}}
+  {{- $storageClasses := dict -}}
+  {{- $defaultStorageClass := "" -}}
+
+  {{- $lookupResult := dict -}}
+  {{- $_ := include "srox.safeLookup" (list $ $lookupResult "storage.k8s.io/v1" "StorageClass" "" "") -}}
+  {{- range $sc := get ($lookupResult.result | default dict) "items" | default list -}}
+    {{- $storageClassName := $sc.metadata.name -}}
+    {{- $annotations := $sc.metadata.annotations | default dict -}}
+    {{- $isDefault := index $annotations "storageclass.kubernetes.io/is-default-class" | default false -}}
+    {{- $_ := set $storageClasses $storageClassName (dict "isDefault" $isDefault) -}}
+    {{- if and $isDefault (not $defaultStorageClass) -}}
+      {{- $defaultStorageClass = $storageClassName -}}
+    {{- end -}}
+  {{- end -}}
+
+  {{ $_ := set $._rox.env.storageClasses "all" (dict "all" $storageClasses) }}
```

Comment:
Maybe I'm missing something but why do we need to add the "all" field here and in the config shape. I don't see where it is being used in this PR.

Reply:
Felt more complete to me.
But you are right, it is currently not used.
I'm open for removal.

Reply:
discussed offline. Might come in handy in the future to have knowledge about existing storageclasses.